cpp/src/community/k_truss_impl.cuh

```diff
@@ -456,27 +456,17 @@ k_truss(raft::handle_t const& handle,
         });
 
       if constexpr (multi_gpu) {
+        std::vector<cugraph::arithmetic_device_uvector_t> edge_properties{};
+
         std::tie(std::get<0>(edgelist_to_update_count),
                  std::get<1>(edgelist_to_update_count),
                  std::ignore,
-                 std::ignore,
-                 std::ignore,
-                 std::ignore,
-                 std::ignore,
                  std::ignore) =
-          detail::shuffle_int_vertex_pairs_with_values_to_local_gpu_by_edge_partitioning<vertex_t,
-                                                                                         edge_t,
-                                                                                         weight_t,
-                                                                                         int32_t,
-                                                                                         int32_t>(
+          detail::shuffle_int_vertex_pairs_with_values_to_local_gpu_by_edge_partitioning(
             handle,
             std::move(std::get<0>(edgelist_to_update_count)),
             std::move(std::get<1>(edgelist_to_update_count)),
-            std::nullopt,
-            std::nullopt,
-            std::nullopt,
-            std::nullopt,
-            std::nullopt,
+            std::move(edge_properties),
```

Comment:
We have `shuffle_ext_edges`. Should we better create `shuffle_int_edges` as well instead of calling this detail space function in the application code?

Reply:
Yes, I meant to do that.  I'll update this in the next push.

Reply:
Added `shuffle_int_edges`